docs/changelog.md

```diff
@@ -2,6 +2,9 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format.
 
+## [0.6.0] -- 2023-08-24
+- Added date filter to project annotation
+- Updated peppy without attmap
```

Comment:
I don't understand what this means. If you updated peppy, shouldn't that be in the peppy changelog?